js/controller.js

```diff
@@ -146,6 +151,9 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       $("#" + elementId + " .player_skin").css("z-index", OO.CSS.ALICE_SKIN_Z_INDEX);
       this.state.mainVideoElement = $("#" + elementId + " .video");
 
+      this.state.playerHeight = $("#" + elementId + " .innerWrapper").height();
```

Comment:
We should probably then change the above statements from jQuery to React as well. I am not sure about how to do it right now.
@chibzu, your thoughts?